.hadolint.yaml

```diff
@@ -0,0 +1,3 @@
+ignored:
+  - DL3008
```

Comment:
Please add comments to explain why these are disabled

Reply:
I actually reverted this, but I'll be sure to add a comment for these in the future if we restore them.